apps/server/src/app/routes/securities.router.ts

```diff
@@ -51,11 +52,28 @@ router.post(
         resolve: async ({ ctx }) => {
             ctx.ability.throwUnlessCan('manage', 'Security')
 
-            await ctx.queueService
-                .getQueue('sync-security')
-                .addBulk([{ name: 'sync-us-stock-tickers', data: {} }])
+            if (env.NX_POLYGON_API_KEY) {
+                await ctx.queueService.getQueue('sync-security').add('sync-us-stock-tickers', {})
+                return { success: true }
+            } else {
+                return { success: false, message: 'No Polygon API key found' }
```

Comment:
@karan0handa should get an error Toast now if you don't have an API key set and click the dev tools

Reply:
Just tested it out. This isn't doing the toast. It's returning a `200` response object because it's successfully returning.

An error needs to be thrown here so that it can be caught by `axios`.